source-code/website/src/pages/index/Header.tsx

```diff
@@ -1,189 +1,108 @@
 import { createSignal, For, Show } from "solid-js";
+import IconClose from "~icons/material-symbols/close-rounded";
+import IconMenu from "~icons/material-symbols/menu-rounded";
+import IconTwitter from "~icons/cib/twitter";
+import IconGithub from "~icons/cib/github";
 
 export function Header() {
 	const links = [
 		{ name: "Docs", href: "/documentation" },
 		{ name: "Editor", href: "/editor" },
 	];
 
-	const [show, setShow] = createSignal(false);
+	const socialMediaLinks = [
+		{
+			name: "Twitter",
+			href: "https://twitter.com/inlangHQ",
+			Icon: IconTwitter,
+		},
+		{
+			name: "Github",
+			href: "https://github.com/inlang/inlang",
+			Icon: IconGithub,
+		},
+	];
+
+	const [mobileMenuIsOpen, setMobileMenuIsOpen] = createSignal(false);
 
 	return (
-		<header class="sticky z-40 top-0  border-b border-outline    ">
-			<nav class=" mx-auto max-w-screen-2xl">
-				<div class=" flex gap-8 p-4 bg-background   ">
+		<header class="sticky z-40 top-0  border-b border-outline">
+			<nav class="mx-auto max-w-screen-2xl">
+				<div class=" flex gap-8 p-4 bg-background">
 					<a href="/" class="flex items-center">
 						<img
 							class="h-8 w-auto "
 							src="/favicon/favicon.ico"
-							alt=" Company Logo"
+							alt="Company Logo"
 						/>
-						<span class=" self-center pl-2 text-xl font-bold">inlang</span>
+						<span class="self-center pl-2 text-xl font-bold">inlang</span>
 					</a>
-					{/* Start icon social media and normal nav   */}
-					<div class="z-10 p-2 grid grid-cols-2 w-full items-center ">
-						<Show when={show() == false}>
-							<div class="hidden md:flex justify-start gap-10">
-								<For each={links}>
-									{(link) => (
-										<a class="link-primary" href={link.href}>
-											{link.name}
-										</a>
-									)}
-								</For>
-							</div>
-
-							{/* start icon social media   */}
-							<div class="hidden md:flex  justify-end gap-10  ">
-								<a href="https://twitter.com/inlangHQ" class="flex space-x-1">
```

Comment:
extracting the social media links into a variable helped to solve a bug in this line. the twitter <a> CSS differed from the github <a>